packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -40,6 +41,9 @@ function SortBy(props) {
 	const onOrderChange = event =>
 		performChange(event, { ...currentValue, isDescending: !currentValue.isDescending });
 
+	if (displayMode === DISPLAY_MODE.TABLE) {
```

Comment:
If you move that to line 12, I will be really happy 👍

Reply:
Not on line 12 because all hooks must run. But line 20 yes

Reply:
done